web/src/assets/customer/empty.svg

```diff
@@ -0,0 +1,112 @@
+<svg width="192" height="150" viewBox="0 0 192 150" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
if it's only used by commit page then call it empty_commits.svg ?

Reply:
@ifhuang HAHA ok I agree that `empty.svg` sounds bad 😆 I feel that `empty_commits.svg` is too specific because the illustration has nothing to do with commits as well. What about `no_content.svg`? Is that weird too? hahaha